.netlify/build.sh

```diff
@@ -23,8 +23,7 @@ mv WORKSPACE.tmp WORKSPACE
 # generation either.
 sed -i 's/vendored_node = "@nixpkgs_nodejs"/vendored_node = None/' WORKSPACE
 
-# XXX: see .bazelrc note [backward compatible options] for rational behind
-# the --incompatible_use_python_toolchains flag
-bazel build --incompatible_use_python_toolchains=false //docs:api_html
+# XXX: Remove --host_force_python=PY2 after updating to Starlark 0.4.0
```

Comment:
What's Starlark 0.4? Is the language now versioned separately from Bazel?

Reply:
Changes below suggest you meant Stardoc.

Reply:
Good catch. Yes, I meant stardoc.